docs_src/microservices/application/services/AppServiceConfigurable/details/AvailablePipelineFunctions.md

```diff
@@ -316,7 +321,25 @@ Please refer to the function's detailed documentation by clicking the function n
         AuthMode: "usernamepassword"
         SecretName: "mqtt"
     ```
-
+    ```yaml
+    # MQTT Export with Will Options
+    MQTTExport:
+      Parameters:
+        BrokerAddress: "tcps://my-broker-host.com:8883"
+        Topic: "mytopic"
+        ClientId: "myclientid"
+        Qos="2"
+        AutoReconnect="true"
+        Retain="true"
+        SkipVerify: "false"
+        PersistOnError: "true"
+        AuthMode: "none"
+        WillEnabled: "true"
+        WillPayload: "serviceX has exited"
+        WillQos: "2"
+        WillRetained: "true"
+        WillTopic: "serviceX/last/will"
+    ```
```

Comment:
Do we want to call this out as a new for EdgeX 3.1?

Reply:
Great suggestion!. Will add that.

Reply:
done